src/global/CMakeLists.txt

```
@@ -0,0 +1,18 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+add_library(k4a_global STATIC 
+            global.c
+            )
+
+# Consumers should #include <k4ainternal/global.h>
+target_include_directories(k4a_global PUBLIC 
+    ${K4A_PRIV_INCLUDE_DIR})
+
+# Dependencies of this library
+target_link_libraries(k4a_global PUBLIC 
```

Should this be PRIVATE instead of PUBLIC? What part of k4ainternal::rwlock do consumers of k4a_global need?

```suggestion
target_link_libraries(k4a_global PRIVATE
```